api/envoy/extensions/filters/http/proto_message_logging/v3/config.proto

```diff
@@ -15,20 +15,26 @@ option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/fil
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 option (xds.annotations.v3.file_status).work_in_progress = true;
 
-// [#not-implemented-hide:]
 // [#protodoc-title: Proto Message Logging]
-// Proto Message Logging :ref:`configuration overview
-// <config_http_filters_proto_message_logging>`.
+//
 // [#extension: envoy.filters.http.proto_message_logging]
 //
-// ProtoMessageLogging filter supports logging scrubbed gRPC requests/responses(proto messages)
-// as google.protobuf.Struct and storing results
-// in the dynamic metadata `envoy.filters.http.proto_message_logging` for later access.
+// Overview
+// --------
+//
+// ProtoMessageLogging filter supports logging scrubbed gRPC
+// requests/responses(proto messages) as google.protobuf.Struct and storing
+// results in the dynamic metadata `envoy.filters.http.proto_message_logging`
```

Comment:
Do we want to call it `proto_message_scrubbing`? I think what it does it to scrub and store in the dynamic metadata. The scrubbed result can be used not only for logging?

Reply:
Makes sense. I modified the filter name to `Proto Message Scrubbing`.

Additionally, modified the `logging` / `auditing` to `scrubbing` in filter config and other places.

Thank you!